community/cloud-foundation/templates/iam_member/iam_member.bats

```diff
@@ -1,3 +1,6 @@
+# THIS IS AN "EXTRA" BATS FILE. THER IS ALREADY ONE IN THE TESTS/INTEGRATION FOLDER.
```

Comment:
Is this file a backup? We should likely clean this up prior to merge

Reply:
@alexander-levitski  Please delete this file. Its redundant..